crates/preset_env_base/Cargo.toml

```diff
@@ -1,23 +1,24 @@
 [package]
-authors = ["강동윤 <kdy1997.dev@gmail.com>"]
-description = "Common logic for targetting vairous browsers"
+authors       = ["강동윤 <kdy1997.dev@gmail.com>"]
+description   = "Common logic for targetting vairous browsers"
 documentation = "https://rustdoc.swc.rs/preset_env_base/"
-edition = "2021"
-license = "Apache-2.0"
-name = "preset_env_base"
-version = "0.3.2"
+edition       = "2021"
+license       = "Apache-2.0"
+name          = "preset_env_base"
+version       = "0.3.2"
 
 [lib]
 bench = false
 
 [dependencies]
-ahash = "0.7.4"
+ahash  = "0.7.4"
 anyhow = "1"
-browserslist-rs = "=0.11.0"
-dashmap = "5.1.0"
-from_variant = { version = "0.1.3", path = "../from_variant" }
-once_cell = "1.12.0"
-semver = { version = "1.0.4", features = ["serde"] }
-serde = { version = "1", features = ["derive"] }
-st-map = "0.1.2"
-tracing = { version = "0.1.32" }
+# https://github.com/browserslist/browserslist-rs/pull/13
+browserslist-rs = { git = "https://github.com/kwonoj/browserslist-rs", rev = "bd47b8e17c937d15ed75fe9c3522ef0ac94ffcd9" }
```

Comment:
This will block publishing

Reply:
ah, I keep forgetting it 🤦 . Guess this is no go and I have to wait https://github.com/browserslist/browserslist-rs/pull/13?

Reply:
Closing for now. as we can't ship it this anyway.